vck-openid-ktor/src/commonTest/kotlin/at/asitplus/wallet/lib/ktor/openid/OpenId4VpWalletTest.kt

```diff
@@ -84,14 +105,129 @@ class OpenId4VpWalletTest : FunSpec() {
oW5ESIhfecXx0Ny8L8pFztCF7toicre4CRBaHe4CIBV1FHLJ+PTLzQmjMnAEuOCkj0uvTow1jIUXDxavsn+3WQgB2BhZB/ynZ3ZlcnNpb25jMS4wb2RpZ2VzdEFsZ29yaXRobWdTSEEtMjU2bHZhbHVlRGlnZXN0c6Fxb3JnLmlzby4xODAxMy41LjG4LgBYIBCirWXR3Res28ECLJHT4eFx1MvGtPHijGVg+p8cKza4AVggWMzCHc/zImRD5uhAlGH3+11ioWO9tvsX33apo2Y2wdMCWCAgw9OT2ZboqBeAAgEu/PbCgGzIc9q1cwkYgmhWJluRsQNYIO+UAfRYiy8s70Abc+cQ8Y0KkJIOcmIoa/Q+ikWY6+Y4BFggzVbHThjw72zTmwIwMOrR6RrjIL4z6/veqSMj0/ndfooFWCCqq3KZcxaFWXKQTHpnfAiCiS4FwsyDWQpUfRtLfuo7HwZYIN16VYbTPbFaeFAYhiti4FkeyyhLRWl9mTVvcQOMwcSHB1gg46OcMO+4Zw6espYRlF7mxdG3Y84Nkzc1cis1hYKcYkEIWCDE8i2E2k0UdId2yAAZYGZq6Bg2J5X5hELxfglEUGujPQlYIN23TK/6tI85WkOV8Be0K3PC4KuyzstTbxD4LBKaxEE4ClggubYa3jgw/nxOX3hhlBa+Ig/G93f9nA9vuEalWMXflIsLWCCN7qIOy2cFYQ12kzi5twH1fmDduohjYoBlAIkoNi5vGwxYIEIfsNhrE1+WkNjkTfrBBGJQ3HYsSqTGmco+G6HpfweNDVggdHw2pb+UWkxs9IdShFhqqJFhnt4Ymf2Ej6zfraexXh8OWCBta/IPgnuPMGdkKHEvVc3+J/pvSJ8edsck2DCY64DSjg9YINT+LoKiYhiTfmHbGc0nKb0laQ0fY7m3a8ZlHrzS+OpCEFggVCbWTAapenLL4ws4AXOB3y5F5XvSM0N5ixlYKZ2X1KcRWCCf4i5JnDhsyZ3px70VQHUpbCgQS2sSdfHO+8y52BauehJYILNdMQb80zUCXVgiFdvIB9oRhOICmyFLTvw69kPVLC0QE1ggH5a6a6OgBRuFpjP3dDUw1dkbsCt8243ljqJ9lJVDjIAUWCAS6uW/SC4gt7SgrTzdVGTEaXwl0vN0RdaEAsgdpEPA3hVYICE9N7mrgW53hCn4SpWaWCGQpPNcV/NDp+MuyUNoWpuUFlggHN1Crix4i8qpUqBnfaK5K6Gl/Hl5pt2qdhaBRfjWwfgXWCAo7wDqO1cNwe3IKW7G421l/eQ/dizy3/mPuCHkrOB0yBgYWCATcaxYfpwn0rlBaayUNgOIxtNrtRpfk/Nm6KFB20eR5RgZWCAASdPaZijImJi1RR4AnOej90r2jsr/45mKwejZX1EAYhgaWCDLarv3qJYh5YVMezxTEIle+DxlgjGnUPzA7vUp0rRU6hgbWCC/0SGmpKBveGdjrGpVCktQWmWWJkH2dNC8YE6AZkcWmxgcWCDyEHVeXEKTKAV2wsEQD+wXZrNrpjbRMpdjRoMi8dkmZBgdWCCogM2mnJ/MdqV56RnZu5fCvMoD3HKEkY80jk7uNClMiBgeWCCRI8u/BmY8z0560/2YPRMaBPZ60MLHukiGZk27vQxTYhgfWCBDWhzSPQIIGcnoYkD2bSIR8PuNzGQVzUX3NKMT5ae5YhggWCBfUp4qSCyJn1YgmrtKB3aZYFgycACA4uCQVs7VY5Cx/RghWCBX0U8yXHNaJM6cBsVjr9ouV6a/ZbFD/tSE/RoParIqCBgiWCDqc2oAD7HUpuV7xbhEgLsbbSMJ/eabk+NFk4vMpgAIpRgjWCATdU2gw4Gji8hPmsFhZx+qFSARdYzSl12hh9jzQ5I+QBgkWCA0Z3Nj3xu9cuNPtAPlbzNsArDQh899fLcCtqrV+NnWrBglWCBf4v08hFXUmbrRLJNmWl5gGoMrjnMFiq3KhPIY4GGoeBgmWCCQv1TKMfiRBQm+3+kD4ZvwE89ieG3/7F2ETyGkPq3vaRgnWCD13qTvXpgKCLcYQCLvqkaoYmo3jBSvnV2ooY/GOIxuJhgoWCA3ChOldqBjI+/vof9K/GockknZN4zuZxqW90KycpxSGBgpWCANavtiellHkq34LGMcbyL91IY9Koy0P3wDg9jXhtq3MBgqWCDx/XVgI8Onhiua4h+Zsa39ChR3gU17shf3C0tNzP6jfxgrWCB/PEakVwIjPHabeF2Jhfz/AtwZ23tShaNHho0f+KR07hgsWCC+/VQTD3GqYq7Bl4B/B0v9y7D2vssrucjgx8X4qYkO/RgtWCD8RjES14jRvQYTeo5iDAetqEycYXZb9QoL7OJK2zB2s21kZXZpY2VLZXlJbmZvoWlkZXZpY2VLZXmkAQIgASFYIJMPYKs3pf7QuBq07JUWD0gfcZ+tCaQLZFXDWhdsQtiyIlgg/6pDXtaq5pLLXemhkFa7332oNTFK70JwYbigwLNtiUdnZG9jVHlwZXVvcmcuaXNvLjE4MDEzLjUuMS5tRExsdmFsaWRpdHlJbmZvo2ZzaWduZWTAeB4yMDI1LTA2LTAyVDExOjM5OjAwLjc1MDI0Mzk2MFppdmFsaWRGcm9twHgeMjAyNS0wNi0wMlQxMTozOTowMC43NTAyNDM5NjBaanZhbGlkVW50aWzAdDIwMjUtMDYtMDlUMTE6Mzk6MDBaZnN0YXR1c6Frc3RhdHVzX2xpc3SiY2lkeBkJtWN1cml4L2h0dHBzOi8vd2FsbGV0LmEtc2l0LmF0L202L2NyZWRlbnRpYWxzL3N0YXR1cy8xWEC2C75R/GxeztKRpiH/tQC/r7+Qd5CI7Vvdy7vWeb61XBBmkqAmLNb1Ljwi6bIS/8LwaPXspjBJfwz61Y8lONI6\"},\"schema-uri\":\"https://wallet.a-sit.at/schemas/1.0.0/MobileDrivingLicence2023.json\"}"
```

Comment:
please remove the byte array with a Base64 encoded string ... or better issue and store the credential in code?!